.github/workflows/tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Tests
+
+on:
+  push:
+    branches: ["develop", "release-*"]
```

Comment:
I don't think we're using Synapse's branching model for this module, are we? Likely enough to set it up to run the workflow on pushes to the `main` branch (and pull requests)

Speaking of, we should probably do the dance to rename `master` to `main` in this repo

Reply:
> I don't think we're using Synapse's branching model for this module, are we? Likely enough to set it up to run the workflow on pushes to the main branch (and pull requests)

Woops, forgot to change that, thanks

> Speaking of, we should probably do the dance to rename master to main in this repo

Huh, I thought I had done that, apparently not, thanks for the heads up

Reply:
Both fixed